### DISCUSSION TITLE
There may be a typo in the jdbc bootstrap `build.gradle.kts` file

### DISCUSSION THREAD

**Discussion by zenvzenv** [open-telemetry/opentelemetry-java-instrumentation#9493] [Q&A]
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/403e133446aabca04e1e465de6f372a2bc0dda32/instrumentation/jdbc/bootstrap/build.gradle.kts#L6

What is `JDDC`, is it mean `JDBC`?

**Comment by laurit** [comment]
Yes it is a typo. I have created https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/9496 to fix it
